cfgov/ask_cfpb/search_indexes.py

```diff
@@ -37,9 +36,42 @@ def prepare_portal_topics(self, obj):
     def prepare_portal_categories(self, obj):
         return [topic.heading for topic in obj.portal_category.all()]
 
+    def extract_raw_text(self, stream_data):
+        # We want answer text to come first, to show up in result snippets.
+        text_chunks = [
+            block.get('value').get('content')
+            for block in stream_data
+            if block.get('type') == 'text'
+        ]
+        extra_chunks = [
+            block.get('value').get('content')
+            for block in stream_data
+            if block.get('type') in ['tip', 'table']
+        ]
+        chunks = text_chunks + extra_chunks
+        return " ".join(chunks)
+
+    def prepare_answer_text(self, obj):
+        stream = obj.answer_content
+        raw_text = self.extract_raw_text(stream.stream_data)
+        return strip_tags(raw_text)
+
+    def prepare_answer_unaccented(self, obj):
+        text = ''
+        if obj.language == 'en':
+            return text
+        stream = obj.answer_content
+        raw_text = self.extract_raw_text(stream.stream_data)
+        stripped = strip_tags(raw_text)
+        try:
+            text = unidecode.unidecode(stripped)
+        except Exception:
+            pass
+        return text
+
     def prepare(self, obj):
         data = super(AnswerPageIndex, self).prepare(obj)
-        data['suggestions'] = data['text']
```

Comment:
What is `data['text']` currently?

Reply:
The "text" field of the search index, which is the bucket of indexed content.

Reply:
Both ways work, so I reverted to data['text'].